packages/forms-web-app/src/pages/projects/register/_translations/translations.test.js

```diff
@@ -0,0 +1,16 @@
+const enCookiesTranslations = require('./en.json');
+const cyCookiesTranslations = require('./cy.json');
+
+describe('pages/cookies/_translations', () => {
+	it('should return the correct English translations', () => {
+		expect(enCookiesTranslations).toMatchSnapshot();
+	});
+
+	it('should return the correct Welsh translations', () => {
+		expect(cyCookiesTranslations).toMatchSnapshot();
+	});
+
+	it('should return the same keys for English and Welsh translations', () => {
```

Comment:
Is this test passing? I have noticed issues where this on my local main where this test is not failing the coverage test but isn't really passing. I think it should look like this:

`it('should returning matching keys for both English and Welsh translations', () => {
    expect(Object.keys(enTranslations).sort()).toStrictEqual(Object.keys(cyTranslations).sort());
});`

Reply:
resolved per discussion - the snapshot testing / testing of nested keys will be addressed in a future ticket